control-plane-operator/controllers/hostedcontrolplane/v2/assets/cluster-autoscaler/podmonitor.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: monitoring.coreos.com/v1
```

Comment:
This needs to be `monitoring.rhobs/v1` because the exported metrics belong to dynatrace and rhobs monitoring and alerting.